Documentation/gettingstarted/ipvlan.rst

```diff
@@ -0,0 +1,112 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    http://docs.cilium.io
+
+.. _ipvlan:
+
+*********************
+Using ipvlan datapath
+*********************
+
+This guide explains how to configure Cilium to set up an ipvlan-based
+datapath instead of the default veth-based one.
+
+.. note::
+
+    This is a beta feature. Please provide feedback and file a GitHub issue
+    if you experience any problems.
+
+First step is to download the Cilium Kubernetes descriptor:
+
+.. tabs::
+  .. group-tab:: K8s 1.13
+
+    .. parsed-literal::
+
+      curl \ |SCM_WEB|\/examples/kubernetes/1.13/cilium.yaml
+
+  .. group-tab:: K8s 1.12
+
+    .. parsed-literal::
+
+      curl \ |SCM_WEB|\/examples/kubernetes/1.12/cilium.yaml
+
+  .. group-tab:: K8s 1.11
+
+    .. parsed-literal::
+
+      curl \ |SCM_WEB|\/examples/kubernetes/1.11/cilium.yaml
+
+  .. group-tab:: K8s 1.10
+
+    .. parsed-literal::
+
+      curl \ |SCM_WEB|\/examples/kubernetes/1.10/cilium.yaml
+
+  .. group-tab:: K8s 1.9
+
+    .. parsed-literal::
+
+      curl \ |SCM_WEB|\/examples/kubernetes/1.9/cilium.yaml
+
+  .. group-tab:: K8s 1.8
+
+    .. parsed-literal::
+
+      curl \ |SCM_WEB|\/examples/kubernetes/1.8/cilium.yaml
+
+Edit the ConfigMap in that file with the etcd server that is running
+in your cluster and set the option ``datapath-mode`` to ``"ipvlan"``.
+It is also required to specify the ipvlan master device which typically
+points to a networking device that is facing the external network. This
+is done through setting ``ipvlan-master-device`` to the name of of the
+networking device such as ``"eth0"``, for example.
```

Comment:
Maybe it's worth mentioning that the device name should be the same on each host.